checkov/terraform/checks/resource/base_resource_value_check.py

```diff
@@ -65,7 +65,7 @@ def scan_resource_conf(self, conf: Dict[str, List[Any]]) -> CheckResult:
             value = dpath.get(conf, inspected_key)
             if isinstance(value, list) and len(value) == 1:
                 value = value[0]
-            if value is None:
+            if value is None or (isinstance(value, list) and not value):
```

Comment:
```suggestion
            if not value:
```

Reply:
I didn't want to do it, because it would also flag `0` or `False` as a missing block case, which would be incorrect. But still good point in general to leverage the truthiness of objects 👍